app/views/container_dashboard/show.html

```diff
@@ -0,0 +1,113 @@
+<div ng-app="containerDashboard" ng-controller="containerDashboardController as dashboard" class="container-fluid container-tiles-pf containers-dashboard">
+    <script type="text/ng-template" id="heatmaps-card.html">
+        <div pf-card class="heatmap-card" head-title={{title}} hidetopborder="{{hideTopBorder}}">
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-12 card-heatmap-body">
+                    <div ng-repeat="heatmap in heatmaps track by $index">
+
+                        <div class="{{columnSizingClass}} container-heatmap-tile">
+                            <span class="h3 heatmap-chart-title" ng-if="heatmaps.length > 1">{{heatmap.title}}</span>
+
+                            <div id="{{heatmap.id}}" class="heatmap-container" style="height: {{heatmapChartHeight}};">
+                                <div heat-map data="heatmap.data" parentid="{{heatmap.id}}" ng-if="heatmap.loadingDone"></div>
+                            </div>
+
+                            <div ng-if="!heatmap.loadingDone" class="spinner spinner-lg loading"></div>
+                        </div>
+                    </div>
+                    <div class="col-xs-16 col-sm-8 col-md-8">
+                        <div class="heatmap-legend heatmap-legend-container">
+                            <div ng-repeat="item in heatMapUsageLegendLabels" class="heatmap-legend-container">
+                                <li class="color-box" style="background-color: {{item.color}};"></li>
+                                <li class="legend-text">{{item.text}}</li>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </script>
```

Comment:
@himdel @AparnaKarve So this is the only method of "templating" managed to get working. Whatever is in the script tag is included in line 90 with ```<div heat-maps-card...``` through the ```heatMapsCard``` directive. 
Though in this case, it might as well be inline since its blowing up this file anyway.